src/gsi/read_ssmi.f90

```diff
@@ -302,13 +303,17 @@ subroutine read_ssmi(mype,val_ssmi,ithin,rmesh,jsatid,gstime,&
 !       SSM/I data are stored in groups of nscan, hence the loop.  
 
         call ufbint(lnbufr,midat,nloc,nscan,iret,str1)
+        if(midat(1,1)>10e8) call ufbint(lnbufr,midat,nloc,nscan,iret,strx)
 
 
 !---    Extract brightness temperature data.  Apply gross check to data. 
 !       If obs fails gross check, reset to missing obs value.
 
         call ufbrep(lnbufr,mirad,1,nchanl*nscan,iret,str2)
 
+        !print*,mirad
```

Comment:
Since this print and abort are commented out, I guess we don't need them.  Is there any reason to keep them in the code as inactive lines?